student.rb

```diff
@@ -2,7 +2,7 @@
 
 class Student < person
```

Comment:
- [ ] I got a `NameError` when I tried running your project. The error is coming from this file. Please look at this and try to work on it. For a piece of advice, it will be nice to verify everything works before asking for another review.

Screenshot |
:--------------:|
![a1](https://user-images.githubusercontent.com/39523312/169653958-26907f5b-ecbb-490a-9a52-a4e56029ad52.png)

Reply:
Thanks @mmsesay , check it up now in the screenshot below 
<img width="405" alt="Screen Shot 2022-05-21 at 16 54 28" src="https://user-images.githubusercontent.com/82823425/169657118-bf1ab335-7262-4914-8615-5232ed272f5a.png">

sorry for the previous mistake.